### PR TITLE
Fixes #8 -- added TLS ALPN challenge support

### LINKED ISSUE #8: Add ALPN challenge support
Right now there's just HTTP and DNS challenges, it'd be great if ALPN challenges were also supported

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: src/order/auth.rs [hunk 1]
```diff
@@ -123,6 +123,20 @@ impl<P: Persist> Auth<P> {
...
+    /// The TLS ALPN challenge is a certificate that must be served when a
+    /// request is made for the ALPN protocol "tls-alpn-01". The certificate
+    /// must contain a single dNSName SAN containing the domain being
+    /// validated, as well as an ACME extension containing the SHA256 of the
+    /// key authorization.
+    pub fn tls_alpn_challenge(&self) -> Challenge<P, TlsAlpn> {
+        self.api_auth
+            .tls_alpn_challenge()
+            .map(|c| Challenge::new(&self.inner, c.clone(), &self.auth_url))
+            .expect("dns-challenge")
```
Comment (algesten) [inline src/order/auth.rs]: tiny typo here

**Review by algesten — APPROVED** [review]
One tiny copy-paste typo to fix. Force push and I merge and release.

Thank you for doing this!

**Comment by alex** [comment]
Done!

**Comment by algesten** [comment]
Released as 0.7.0. Thanks!

**Pull request merged**